### DISCUSSION TITLE
Touch screen (multi-touch) support for Lazer

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#15708] [Ideas] [answered]
Currently Lazer only recognizes the first touch, which means if you alternate to a second touch while the first touch is still active it will ignore it completely. Stable deals with this just fine by recognizing the second (and further consecutive taps) as an alternation of the mouse and also moves the cursor to the most recent tap location. I've seen a discussion about general touchscreen support but I am making a separate one for multi touch. My playstyle centers around the touch input api and alternation when both devices are tapping (I play with tablet emulating touch input, combined with an actual touchscreen with my other hand.)

**Comment by peppy** [comment, marked as answer]
it's the same as other discussions, doesn't require a new one.
